native/src/constants/dimensions.ts

```diff
@@ -1,17 +1,25 @@
+import { Dimensions } from 'react-native'
+
 export type DimensionsType = {
   headerHeight: number
   modalHeaderHeight: number
   categoryListItem: {
     iconSize: number
     margin: number
   }
+  fontScaling: number
+  deviceWidth: number
+  headerTextSize: number
 }
 const dimensions: DimensionsType = {
   headerHeight: 60,
   modalHeaderHeight: 40,
   categoryListItem: {
     iconSize: 40,
     margin: 10
-  }
+  },
+  fontScaling: 0.04,
+  deviceWidth: Number(Dimensions.get('window').width),
```

Comment:
This can change when you rotate your device. I think there are simple implementations of hook around which allow automatic updating of this value.

Reply:
thx for info didn't know about that hook :) works fine